src/commands/pull-image.yml

```diff
@@ -0,0 +1,25 @@
+description: Pull a container image to the Amazon ECR registry
+
+parameters:
+  account-url:
+    type: env_var_name
+    description: >
+      Env var storing Amazon ECR account URL that maps to an AWS account,
+      e.g. {awsAccountNum}.dkr.ecr.us-west-2.amazonaws.com
+      defaults to AWS_ECR_ACCOUNT_URL
+    default: AWS_ECR_ACCOUNT_URL
+
+  repo:
+    type: string
+    description: Name of an Amazon ECR repository
+
+  tag:
+    description: A comma-separated string containing docker image tags (default = latest)
+    type: string
+    default: "latest"
+
+steps:
+  - run:
+      name: Pull image to Amazon ECR
+      command: |
+        docker pull $<<parameters.account-url>>/<<parameters.repo>>:${tag}
```

Comment:
tag here is looking for an env var rather than a parameter. This should be changed to include. <<>>. The description also makes mention of a comma-separated list of tags. Could you describe the intent?